.detekt.yml

```diff
@@ -25,6 +25,8 @@ formatting:
     active: false
   MaximumLineLength:
     active: false
+  NoUnusedImports:
+    active: false
```

Comment:
Does enabling this automatically disable the ktlint rule? Because I expected some ktlint related change in this commit.

Reply:
This *is* the `ktlint`-related change: All of `detekt`'s formatting rules just delegate to `ktlint` which is used as a plugin. This commit disables the `detekt` wrapper around the `ktlint` rule, as `detekt` has the same check as a "style" check. I.e., before this change, missing imports were reported twice since the upgrade to `detekt` 1.8.0, once as a `detekt` style issue, and once as a `ktlint` formatting issue.

Reply:
Thanks, I somehow missed that you set it to `false`.